chainer/functions/connection/convolution_2d.py

```diff
@@ -92,7 +92,9 @@ def forward_gpu(self, inputs):
                 self.bias_desc = cudnn.create_tensor_descriptor(
                     b[None, :, None, None])
 
-            self.max_workspace_size = c * kh * kw * 4
+            self.max_workspace_size = max(
+                out_c * c * 64,  # for winograd
```

Comment:
This statement decide maximum working space size for chose of the best algorithm.
It will be fix in #1122 because this implementation is not good.